src/mmw/apps/modeling/models.py

```diff
@@ -70,9 +75,9 @@ class Project(models.Model):
         help_text='Whether or not this project currently uses a custom weather'
                   ' dataset. If true, requires a non-NULL value for'
                   ' custom_weather_dataset.')
-    custom_weather_dataset = models.CharField(
+    custom_weather_dataset = FileField(
         null=True,
-        max_length=511,
+        upload_to=project_filename,
         help_text='The S3 key for the custom weather dataset.')
```

Comment:
May want to update the `help_text` to reflect the new understanding of the field's content. Although, perhaps it still is ultimately the s3 key.

Reply:
Ah good catch! Fixed in bb1caa5.